source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -93,45 +93,54 @@ struct ImageSetConfig
     static ImageSetConfig Deserialize(const Json::Value& json, const ImageSetConfig& defaultValue);
 };
 
-struct ContainerStyleConfig
+struct AdaptiveCardConfig
 {
-    std::string backgroundColor = "#00FFFFFF";
-    std::string borderColor = "#00FFFFFF";
-    SpacingDefinition borderThickness;
-    SpacingDefinition padding;
+    SpacingDefinition padding = { 8, 8, 8, 8 };
+    bool allowCustomStyle = true;
 
-    static ContainerStyleConfig Deserialize(const Json::Value& json, const ContainerStyleConfig& defaultValue);
+    static AdaptiveCardConfig Deserialize(const Json::Value& json, const AdaptiveCardConfig& defaultValue);
 };
 
-struct ContainerConfig
+struct FactSetConfig
 {
-    ContainerStyleConfig normal;
-    ContainerStyleConfig emphasis = { "#FFEEEEEE", "#FFAAAAAA", SpacingDefinition{ 1, 1, 1, 1 }, SpacingDefinition{10, 10, 10, 10 } };
+    TextConfig title = { TextWeight::Bolder };
+    TextConfig value;
+    unsigned int spacing = 20;
 
-    static ContainerConfig Deserialize(const Json::Value& json, const ContainerConfig& defaultValue);
+    static FactSetConfig Deserialize(const Json::Value& json, const FactSetConfig& defaultValue);
 };
 
-struct AdaptiveCardConfig
+struct ContainerStyleDefinition
 {
-    SpacingDefinition padding = { 8, 8, 8, 8 };
     std::string backgroundColor = "#FFFFFFFF";
+    ColorsConfig foregroundColors;
```

Comment:
I think this should be "fontColors"?

Reply:
Per a conversation this morning we have agreed to rename fontColors into foregroundColors as these colors might apply to more than just fonts in the future.

Reply:
thanks